### ISSUE TITLE
Weird display of GUID (RAW(16)) type (Oracle)

### ISSUE THREAD

**Issue by plotn** [dbeaver/dbeaver#18091]
### Description

as shown...

### DBeaver Version

22.2.0 CE

### Operating System

win 10

### Database and driver

Oracle[
<img width="695" alt="dbeaver_rowid" src="https://user-images.githubusercontent.com/11477081/199187937-5da5b46d-5ee8-4456-81d9-4a94139185ea.png">
](url)

**Comment by plotn** [comment]
See:
SELECT sys_guid(), rawtohex(sys_guid()) FROM dual
for testing

**Comment by E1izabeth** [comment]
You can set `Value display format` to `Database native`. Maybe it's similar to the requested representation.

![image](https://user-images.githubusercontent.com/28875055/199498193-3e35ff18-a962-42c6-ada1-e3ee33ab7b86.png)

**Comment by plotn** [comment]
> You can set `Value display format` to `Database native`. Maybe it's similar to the requested representation.

Yes, It helps. But it is very non-obvious.... I dont know, maybe the issue is not needed anymore, what do you think?

**Comment by E1izabeth** [comment]
We discussed that we could try improving guid representation by default. Let's keep this issue open. Thank you for report.

**Comment by mkgrgis** [comment]
This also applicable to BLOB(16) in SQLite.

**Comment by mkgrgis** [comment]
More recent form for the same problem (DBeaver 23.1.2)
Attributes
![1](https://github.com/dbeaver/dbeaver/assets/41448637/acb89cd2-b187-4517-ba2b-7690eedda8cd)
Data grid output
![2](https://github.com/dbeaver/dbeaver/assets/41448637/1f679c2b-6f8e-4b5d-a7ee-2c2430d34e47)

**Comment by brytkhalifa** [comment]
I encountered the same problem until I stumbled upon this issue. The fix isn't immediately apparent.
